quickstarts/etcd/config.yml

```diff
@@ -0,0 +1,54 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: etcd
+
+# Displayed in the UI (required)
+title: etcd
```

Comment:
I'm not sure about this particular product, but do we want to capitalize this first letter here? I noticed its capitalized in the dashboard name.

Reply:
`etcd` is all lowercase on their site.  I can capitalize it if we feel it would look better.

https://etcd.io/

Reply:
I went ahead and capitalized `Etcd` where necessary.